keyboard.js

```diff
@@ -0,0 +1,6 @@
+const View = require('./view');
+
+class Keyboard {
+}
+
+module.exports = Keyboard;
```

Comment:
Like alternative you can use `import {View} from './view'` instead `const View = require('./view');`.
Also `export class Keyboard {
 }`.